schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,52 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
Same for the others. 

---
In reply to: [124428113](https://github.com/Azure/azure-resource-manager-schemas/pull/326#discussion_r124428113) [](ancestors = 124428113)